README.md

```diff
@@ -0,0 +1,2 @@
+# http_mptcp
+This 
```

Comment:
can you remove the binary files from the `dist` directory? (and add a `.gitignore` file?)